web-common/src/features/exports/ExportMenu.svelte

```diff
@@ -37,14 +38,21 @@
   }
 
   const exportDash = createQueryServiceExport();
-  const { reports } = featureFlags;
+  const { reports, adminServer, exportMetadata } = featureFlags;
 
-  async function handleExport(format: V1ExportFormat) {
+  async function handleExport(format: V1ExportFormat, includeHeader = false) {
```

Comment:
I don't like boolean flag function parameters. It makes the call site hard to read. E.g. when I see `handleExport(V1ExportFormat.EXPORT_FORMAT_CSV, true)}`, I have no way of knowing what `true` indicates unless I read the function signature. Instead, how about we use an options object here:

```ts
async function handleExport(options: { format: V1ExportFormat; includeHeader?: boolean }) {
  const { format, includeHeader = false } = options;
  // ...rest of the code
}
```
...which is more clear at the call site:
```ts
handleExport({ format: V1ExportFormat.EXPORT_FORMAT_CSV, includeHeader: true });
```